fhirAssembly/src/main/resources/assembly/com.vantiq.fhir.fhirConnection/procedures/com/vantiq/fhir/fhirService_create.vail

```diff
@@ -0,0 +1,35 @@
+package com.vantiq.fhir
+PROCEDURE fhirService.create(type String REQUIRED, resource Object REQUIRED, versionId String, generalParms Object): Object
+
+var path = type
+
+if (generalParms) {
```

Comment:
nit: indents look a little funny. i think there are tabs on most lines, but spaces on 1 or 2 (occurs in all the VAIL files here).

Reply:
ran `expand -t 4` & fixed these, I think.